submissions/karelskiy/js-DOM/script.js

```diff
@@ -0,0 +1,83 @@
+  constructor(model, description, cost, img) {
+    this.model = model;
+    this.description = description;
+    this.cost = cost;
+    this.img = img
+  }
+}
+
+const phone = [
+  new iPhone(
+    "iPhone 11 Pro",
+    "6.5-inch Super Retina XDR OLED display. Water and dust resistant (4 meters for up to 30 minutes, IP68). Triple-camera system with 12MP Ultra Wide, Wide, and Telephoto cameras; Night mode, Portrait mode, and 4K video up to 60fps. Face ID for secure authentication and Apple Pay. A13 Bionic chip with third-generation Neural Engine. Fast charge with 18W adapter included. Wireless charging",
+    "Starting at $999",
+    "./img/proM.jpeg"
+  ),
+
+  new iPhone(
+    'iPhone 11',
+    '6.1-inch Liquid Retina HD LCD display. Water and dust resistant (2 meters for up to 30 minutes, IP68). Dual-camera system with 12MP Ultra Wide and Wide cameras; Night mode, Portrait mode, and 4K video up to 60fps. 12MP TrueDepth front camera with Portrait mode, 4K video, and Slo-Mo. Face ID for secure authentication and Apple Pay. A13 Bionic chip with third-generation Neural Engine. Fast-charge capable. Wireless charging.',
+    'Starting at $699',
+    './img/x11.jpeg'
+
+  ),
+  new iPhone(
+    "iPhone Xr",
+    '6.1-inch Liquid Retina display (LCD). IP67 water and dust resistant (maximum depth of 1 meter up to 30 minutes). 12MP camera with OIS and 7MP TrueDepth front camera—Portrait mode, Portrait Lighting, Depth Control, and Smart HDR. Face ID for secure authentication. A12 Bionic with next-generation Neural Engine. Wireless charging—works with Qi chargers. iOS 12 with Memoji, Screen Time, Siri Shortcuts, and Group FaceTime.',
+    'Starting at $649',
+    './img/x.png'
+  ),
+
+  new iPhone(
+    'iPhone 8',
+    '4.7-inch Retina HD display with True Tone. IP67 water and dust resistant (maximum depth of 1 meter up to 30 minutes). 12MP camera with OIS and 4K video. 7MP FaceTime HD camera with Retina Flash. Touch ID for secure authentication. A11 Bionic with Neural Engine. iOS 12 with Screen Time, Group FaceTime, and even faster performance. ',
+    'Starting at $449',
+    './img/eight.jpeg'
+  )
+]
+
+const ul = document.querySelector('.ul-menu');
+const container = document.querySelector('.container');
+
+
+const add = ul.addEventListener('click', (event) => {
+
+  const fragment = document.createElement('div');
+  fragment.classList.add('likeafragment');
+
+
+  if (event.target.nodeName === 'LI' || event.target.nodeName === 'A') {
+    if (container.firstElementChild) container.firstElementChild.remove();
+    phone.forEach((el) => {
+      if (event.target.textContent == el['model']) {
+        const contText = document.createElement('div');
+        const description = document.createElement('div');
+        const cost = document.createElement('p');
+        const img = document.createElement('img');
+
+        // split descriptions by .
+        const array = el['description'].split('. ');
+        for (let i = 0; i < array.length; i++) {
```

Comment:
For simple iteration use [forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) method instead of loop `for`.